components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/servlet/OIDCLogoutServlet.java

```diff
@@ -814,11 +839,11 @@ private boolean getOpenIDConnectSkipUserConsent(HttpServletRequest request) thro
     /**
      * Sends logout token to registered back-channel logout uris.
      *
-     * @param request
+     * @param opbsCookieValue
      */
-    private void doBackChannelLogout(HttpServletRequest request) {
+    private void doBackChannelLogout(String opbsCookieValue) {
 
-        LogoutRequestSender.getInstance().sendLogoutRequests(request);
+        LogoutRequestSender.getInstance().sendLogoutRequests(opbsCookieValue);
```

Comment:
We are passing String instead of HttpServletRequest object, Can't this cause an issue in callee?

Reply:
LogoutRequestSender.getInstance().sendLogoutRequests() has multiple implementations where it can accept a String value as well.

Reply:
ACK